src/hci/gnomex/controller/SaveRequest.java

```diff
@@ -981,7 +981,7 @@ public Command execute() throws GNomExRollbackException {
 			throw e;
 		} catch (ProductException e) {
 			log.error("An exception has occurred in SaveRequest ", e);
-			log.error("Unable to create ProductLedger for request. " + e.getMessage());
+			log.error("Unable to create ProductLedger for request. " + e.getMessage(), e);
 			e.printStackTrace();

```

Comment:
This should be avoided.  It will produce entries in the log without any of the context the logger provides. Only appropriate, IMHO, maybe...when you're trying to debug something during active dev.  Should never make it into production code.